RELEASENOTES-1.4.docu

```diff
@@ -469,4 +469,9 @@
       This syntax can't be used for variadic arguments or any argument
       corresponding to a method parameter declared `inline`
       <bug number="SIMICS-20473"/>.</add-note></build-id>
+  <build-id value="next"><add-note>Fixed a bug where a top-level
+      object named the same as the device object could result in
+      invalid generated C when both objects are targets of <tt>each
+      .. in ..</tt> expressions
+      <bug number="SIMICS-21044"/>.</add-note></build-id>
```

Comment:
Curious line break. How come you didn't do
```
      .. in ..</tt> expressions <bug number="SIMICS-21044"/>.
  </add-note></build-id>
```
?

Reply:
That's a convention I follow for merge conflict resolution reasons: if two notes are added in simultaneous PRs to the same file, and both add identical `</add-note></build-id>` lines, then merge tools will tend to merge those lines. So if you always attach the </add-note> to the previous word, then your note will never be part of an unpleasant merge.

Reply:
I had guessed as much. Wasn't sure it would trump having line breaks be at regular points, but it doesn't matter to me.